awx/main/tests/live/tests/test_host_update_contention.py

```diff
@@ -0,0 +1,81 @@
+import multiprocessing
+import random
+
+from django.db import connection
+from django.utils.timezone import now
+
+from awx.main.models import Inventory, Host
+from awx.main.utils.db import bulk_update_sorted_by_id
+
+
+def worker_delete_target(ready_event, continue_event, field_name):
+    """Runs the bulk update, will be called in duplicate, in parallel"""
+    inv = Inventory.objects.get(organization__name='Default', name='test_host_update_contention')
+    host_list = list(inv.hosts.all())
+    # Using random.shuffle for non-security-critical shuffling in a test
+    random.shuffle(host_list)  # NOSONAR
+    for i, host in enumerate(host_list):
+        setattr(host, field_name, f'my_var: {i}')
+
+    # ready to do the bulk_update
+    print('worker has loaded all the hosts needed')
+    ready_event.set()
+    # wait for the coordination message
+    continue_event.wait()
+
+    # # presumed fix
+    # host_list = sorted(host_list, key=lambda host: host.id)
```

Comment:
You should be able to remove this comment now, since the test now uses `bulk_update_sorted_by_id` and the dev-env check is passing. The test output shows it ran the 1 test in this file.